lib/plugins/create/templates/azure-nodejs/serverless.yml

```diff
@@ -17,12 +17,55 @@ service: azure-nodejs # NOTE: update this with your service name
 # Check out our docs for more details
 # frameworkVersion: "=X.X.X"
 
+# if you are using webpack for packaging
+# custom:
+#   webpack:
+#     keepOutputDirectory: true
+
 provider:
   name: azure
-  location: West US 2
+  region: West US 2
+  runtime: nodejs10.x
+  # prefix: "sample"  # prefix of generated resource name
+  # subscriptionId: A356AC8C-E310-44F4-BF85-C7F29044AF99
+  # stage: dev
+  # type: premium  # premium azure functions
+
+  environment: # these will be created as application settings
+    VARIABLE_FOO: 'foo'
+
+  # you can define apim configuration here
+  # apim:
+  #   api:
+  #     name: v1
+  #     subscriptionRequired: false
+  #     displayName: v1
+  #     description: V1 sample app APIs
+  #     protocols:
+  #       - https
+  #     path: v1
+  #     tags:
+  #       - tag1
+  #       - tag2
+  #     authorization: none
+  #   cors:
+  #     allowCredentials: false
+  #     allowedOrigins:
+  #       - "*"
+  #     allowedMethods:
+  #       - GET
+  #       - POST
+  #       - PUT
+  #       - DELETE
+  #       - PATCH
+  #     allowedHeaders:
+  #       - "*"
+  #     exposeHeaders:
+  #       - "*"
 
 plugins:
   - serverless-azure-functions
+  # - serverless-webpack  # to use webpack for packaging code
```

Comment:
Not sure if we should add the `serverless-webpack` dependency, because otherwise users might run into problems when they uncomment this (I think I'm -1 for that since it'll bloat the template).

Another alternative would be to add a comment that they need to install the `npm` dependency if they uncomment it.

Reply:
I prefer the alternative with adding a comment.  People might not be aware there's a webpack plugin for serverless.

Reply:
Problem with this, is that ucommenting this, will not make it work. It'll actually crash the deploy as user has to do other (not documented here things) to successfully rely on a plugin.

It's also the reason why we don't do it in other templates. There's dedicated https://github.com/serverless/plugins repository which lists all the valuable plugins and links to their documentation.

I'd suggest that in main readme we either link to some external page that lists all valuable plugins that can be used with Azure, or simply list them there.

Reply:
I'd vote for replacing the webpack array element here with a comment that it can be used and where one can find it.

Reply:
done, replace with comment linking to plugins repo